mysql/infoschema.go

```diff
@@ -375,6 +380,42 @@ func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (forei
 	return foreignKeys, nil
 }
 
+// getIndexes return list all the indexes.
```

Comment:
Suggest:

// getIndexes return a list all indexes for the specified table.

Reply:
updated comment